tpm2tools/flush_active_handles_test.go

```diff
@@ -0,0 +1,100 @@
+package tpm2tools
+
+import (
+    "crypto/rand"
+    "crypto/rsa"
+    "testing"
+
+    "github.com/google/go-tpm/tpm2"
+
+    "github.com/google/go-tpm-tools/simulator"
+)
+
+const (
+    // How many keys/handles can the simulator contain at once.
+    maxHandles = 3
+)
+
+func loadRandomExternalKey(t *testing.T, simulator *simulator.Simulator) {
```

Comment:
As @awly said, clearing all transient keys is best put in as a standalone binary. The function to list all handles of a specific type; however, should be a library function.

Reply:
Done.